src/dataframerow/dataframerow.jl

```diff
@@ -62,11 +62,13 @@ Base.values(r::DataFrameRow) = ntuple(col -> parent(r)[col][row(r)], length(r))
 
 # hash column element
 Base.@propagate_inbounds hash_colel(v::AbstractArray, i, h::UInt = zero(UInt)) = hash(v[i], h)
-Base.@propagate_inbounds hash_colel(v::AbstractCategoricalArray, i, h::UInt = zero(UInt)) =
```

Comment:
why is this not needed any more?

Reply:
Woops, I forgot to widen the signature of the following method.